flashcards/backend/decks/tests/test_decks.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+
+from decks.factories import CategoryFactory, DeckFactory, FlashcardFactory, TagFactory, DifficultyLevelFactory
+
+
+class TestFactories(TestCase):
+    def test_category_factory(self):
+        """Test CategoryFactory."""
+        category = CategoryFactory()
+        self.assertIsNotNone(category)
+        self.assertIsNotNone(category.name)
+        self.assertIsNotNone(category.description)
+
```

Comment:
@BartekSzymik what @MWasile is saying is that you use UnitTest for testing and our standard is pytest. Those tests are only passing because pytest can run also UnitTest cases. You need rewrite this file using pytest package and not UniTest. I will only say that as we can build class-based tests with pytest, it is strongly recommended to use functions for testing ;)

Reply:
Thank you Martyna